### PR TITLE
Revert to using main for focalboard

### PR DESCRIPTION
**Pull request by agnivade**
```release-note
NONE
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
           cd ..
           git clone --depth=1 --no-single-branch https://github.com/mattermost/focalboard.git
           cd focalboard
-          git checkout $CIRCLE_BRANCH || git checkout rolling-stable
+          git checkout $CIRCLE_BRANCH || git checkout main
```
Comment (hanzei) [inline .circleci/config.yml:34]: Does it make sense to configure the default branch i.e.`main` via an env variable that is set in CircleCI so we can switch back and forth more easily?

**Review by hanzei — COMMENTED** [review]
0/5 idea

File: .circleci/config.yml (hunk 1 above)
Comment (agnivade) [reply .circleci/config.yml:34]: I don't see a need to back and forth really. So there is no need to keep it variable.

**Review by isacikgoz — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (wiggin77) [reply .circleci/config.yml:34]: We cannot be shipping main.  The issue here is we need to handle breaking changed in mm-server in a fashion that all repos merge into the correct place at the same time.

**Pull request closed without merging**